mailbox/cassandra/pom.xml

```diff
@@ -153,26 +153,4 @@
         </dependency>
     </dependencies>
 
-    <profiles>
-        <profile>
-            <id>noTest</id>
-            <activation>
-                <os>
-                    <family>windows</family>
```

Comment:
great! but did you check?

Reply:
For what ? We will put Assume to disable failing tests under windows when people will complain. I don't see that as a problem because : 
1. people don't run tests on their computers
2. people don't use windows anyway
3. even windows people have docker now